consensus/node.go

```diff
@@ -53,12 +53,6 @@ func (n *Node) Start(ethRPCClient *ethRpc.Client, db *database.Database) error {
 		return err
 	}
 
-	n.logger.Debug("Initializing consensus state...")
```

Comment:
What happened with this? I cannot see where you moved

Reply:
to the constructor instead of using evil setters.